docs/sources/operators-guide/run-production-environment-with-helm/_index.md

```diff
@@ -0,0 +1,284 @@
+       alertmanager_storage:
+         s3:
+           bucket_name: mimir-alertmanager
+       ruler_storage:
+         s3:
+           bucket_name: mimir-ruler
+
+       # The following admin_client configuration only applies to Grafana Enterprise Metrics deployments:
+       #admin_client:
+       #  storage:
+       #    s3:
+       #      bucket_name: gem-admin
+   ```
+
+## Meet security compliance regulations
+
+Grafana Mimir does not require any special permissions on the hosts that it
+runs on. Because of this, you can deploy it in environments that enforce the
+Kubernetes [Restricted security policy](https://kubernetes.io/docs/concepts/security/pod-security-standards/).
+
+In Kubernetes v1.23 and higher, the Restricted policy can be enforced via a
+namespace label on the Namespace resource where Mimir is deployed. For example:
+
+```
+pod-security.kubernetes.io/enforce: restricted
+```
+
+In Kubernetes versions prior to 1.23, the `mimir-distributed` Helm chart
+provides a [PodSecurityPolicy resource](https://v1-24.docs.kubernetes.io/docs/concepts/security/pod-security-policy/)
+that enforces many of the recommendations from the Restricted policy that the
+namespace label enforces.
+To enable the PodSecurityPolicy admission controller for your Kubernetes
+cluster, refer to
+[How do I turn on an admission controller?](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#how-do-i-turn-on-an-admission-controller).
+
+## Monitor the health of your Grafana Mimir cluster
+
+To monitor the health of your Grafana Mimir cluster, which is also known as
+_metamonitoring_, you can use ready-made Grafana dashboards, and Prometheus
+alerting and recording rules.
+For more information, see [Installing Grafana Mimir dashboards and alerts]({{<relref "../monitor-grafana-mimir/installing-dashboards-and-alerts/">}}).
+
+The `mimir-distributed` Helm chart makes it easy for you to collect metrics and
+logs from Mimir. It assigns the correct labels for you so that the dashboards
+and alerts simply work. The chart uses the Grafana Agent to ship metrics to
+a Prometheus-compatible server and logs to a Loki or GEL (Grafana Enterprise
+Metrics) server.
+
+1. Download the Grafana Agent Operator Custom Resource Definitions (CRDs) from
+   https://github.com/grafana/agent/tree/main/production/operator/crds
+2. Install the CRDs on your cluster:
+
+   ```bash
+   kubectl apply -f production/operator/crds/
+   ```
+
+3. Add the following YAML snippet to your
+   values file, to send metamonitoring telemetry from Mimir:
+
+   ```yaml
+   metaMonitoring:
+     serviceMonitor:
+       enabled: true
+     grafanaAgent:
+       enabled: true
+       installOperator: true
+
+       logs:
+         remote:
+           url: "https://example.com/loki/api/v1/push"
+           auth:
+             username: 12345
+
+       metrics:
+         remote:
+           url: "https://mimir-nginx.mimir.svc.cluster.local./api/v1/push"
```

Comment:
We recommend sending it elsewhere, not in the mimir cluster itself. I will correct this with a different URL and mention that sending it locally is achieved with an empty URL

Reply:
Ack. Actually we should not even mention sending locally since in production you need something independent. I would advertise sending the metamonitoring data into Grafana.com primarily and a local Prometheus or second Mimir on-prem.